### DISCUSSION TITLE
cpp/object_oriented/polymorphism_in_cplusplus

### DISCUSSION THREAD

**Discussion by giscus** [imarvinle/CSGuide#34] [Announcements]
# cpp/object_oriented/polymorphism_in_cplusplus

编程指北: 计算机系统学习指南, 操作系统, 计算机网络, C++, Java, 算法, 数据结构

https://csguide.cn/cpp/object_oriented/polymorphism_in_cplusplus.html

**Comment by Xlinford** [comment]
关于函数重载可以看下这篇文章 重载、重写、隐藏的区别 链接放成了local地址
